bedrock/mozorg/templates/mozorg/about/leadership/includes/boards.html

```diff
@@ -348,6 +348,24 @@ <h2 class="group-title" id="board-foundation">Mozilla Foundation</h2>
       </div>
     </article>
 
+    <article id="zain-habboo" data-id="zain-habboo" class="vcard" itemscope itemtype="http://schema.org/Person">
+      <figure class="headshot">
+        {{ resp_img(
+          url='img/mozorg/about/leadership/zain-habboo.jpg',
+          srcset={
+            'img/mozorg/about/leadership/zain-habboo-high-res.jpg': '1.5x'
```

Comment:
Looks like the source image is spelled incorrectly. should be `zain` instead of `sain`

Reply:
🤦